src/test/datascience/errorHandler.unit.test.ts

```diff
@@ -231,7 +231,7 @@ suite('DataScience Error Handler Unit Tests', () => {
 
             const expectedMessage = DataScience.failedToStartKernelDueToImportFailureFromFile().format(
                 'Random',
-                getDisplayPath('c:\\Development\\samples\\pySamples\\sample1\\kernel_issues\\start\\random.py', [])
```

Comment:
I think we should discuss this in standup, we shouldn't be using `path-browserify` in node (desktop), why not use the native `path` module instead of a replacement. This is what webpack should do, in web, use `path-browserify` and in desktop use native `path`, instead of always using a custom module instead of natives.

Reply:
Based on the usage for `path-browserify` you don't need to do what you're doing here, their recommendation is to continue using `path` directly
https://www.npmjs.com/package/path-browserify#install

Reply:
Yeah for the errorUtils case, maybe this is a exception. It could have webpack determine which path to use.

Although that test was wrong as the code it was testing never called getDisplayPath.

Reply:
Not sure what we're talking about now, i'm referring to usage documentation

Reply:
It's explicit so that the users of that code realize what's happening. The problem with using path-browserify in the web is that you get unintended side effects